src/black_primer/primer.json

```diff
@@ -53,8 +56,6 @@
       "py_versions": ["all"]
     },
     "pandas": {
-      "disabled_reason": "black-primer runs failing on Pandas - #2193",
-      "disabled": true,
       "cli_arguments": ["--experimental-string-processing"],
```

Comment:
Hmm, my local copy of Black from main failed to format a single file for pandas. Looks like either my copy of mypyc is subtly broken or my source files are out of date.